Extension/src/common.ts

```diff
@@ -557,6 +558,61 @@ export function getInstalledBinaryPlatform(): string | undefined {
     return installedPlatform;
 }
 
+/* Check if the core binaries exists in extension's installation folder */
+export async function checkInstallBinariesExist(): Promise<boolean> {
+    if (!checkInstallLockFile()) {
+        return false;
+    }
+    let installBinariesExist: boolean = true;
+    const info: PlatformInformation = await PlatformInformation.GetPlatformInformation();
+    const packageManager: PackageManager = new PackageManager(info);
+    const packages: Promise<IPackage[]> = packageManager.GetPackages();
+    for (const pkg of await packages) {
+        if (pkg.binaries) {
+            await Promise.all(pkg.binaries.map(async (file: string) => {
+                if (!await checkFileExists(file) &&
+                    !(info.platform === "win32" && await checkFileExists(file + ".exe"))) {
```

Comment:
This additional check seems unnecessary if the entry in the package.json is changed.